docs/how-to-write-a-userstyle.md

```diff
@@ -0,0 +1,208 @@
+    @frappe:    { @rosewater: #f2d5cf; @flamingo: #eebebe; @pink: #f4b8e4; @mauve: #ca9ee6; @red: #e78284; @maroon: #ea999c; @peach: #ef9f76; @yellow: #e5c890; @green: #a6d189; @teal: #81c8be; @sky: #99d1db; @sapphire: #85c1dc; @blue: #8caaee; @lavender: #babbf1; @text: #c6d0f5; @subtext1: #b5bfe2; @subtext0: #a5adce; @overlay2: #949cbb; @overlay1: #838ba7; @overlay0: #737994; @surface2: #626880; @surface1: #51576d; @surface0: #414559; @base: #303446; @mantle: #292c3c; @crust: #232634; };
+    @macchiato: { @rosewater: #f4dbd6; @flamingo: #f0c6c6; @pink: #f5bde6; @mauve: #c6a0f6; @red: #ed8796; @maroon: #ee99a0; @peach: #f5a97f; @yellow: #eed49f; @green: #a6da95; @teal: #8bd5ca; @sky: #91d7e3; @sapphire: #7dc4e4; @blue: #8aadf4; @lavender: #b7bdf8; @text: #cad3f5; @subtext1: #b8c0e0; @subtext0: #a5adcb; @overlay2: #939ab7; @overlay1: #8087a2; @overlay0: #6e738d; @surface2: #5b6078; @surface1: #494d64; @surface0: #363a4f; @base: #24273a; @mantle: #1e2030; @crust: #181926; };
+    @mocha:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b; };
+}
+```
+
+### The `#catppuccin` mixin
+
+We'll refer to this next section of the template as the "`#catppuccin` mixin". You can ignore the first half of this section with all of the `@<color>: @catppuccin[@@lookup][@<color>];` lines and skip to the comment where it says `// Start styling your website here:`. As the comment suggests, you will write the CSS rules for your port here. Here is what it looks like for our `example.org` port:
+
+<!-- prettier-ignore -->
+```less
+#catppuccin(@lookup, @accent) {
+  @rosewater: @catppuccin[@@lookup][@rosewater];
+  /* ... */
+  @accent-color: @catppuccin[@@lookup][@@accent];
+
+  body {
+    background-color: @surface0;
+
+    & > div {
+      color: @text;
+      background-color: @base;
+    }
+  }
+
+  a:link, a:visited {
+    color: @blue;
+  }
+
+  /* ... */
+```
+
+#### Applying the theme
+
+The final section of the template is about applying the user's light/dark flavors. Read the comments thoroughly and decide which of the two options works best for your port. `example.org` doesn't have a light/dark mode toggle, so we'll apply it based on the user's preferences.
+
+<!-- prettier-ignore -->
+```less
+@media (prefers-color-scheme: light) {
+  :root {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+}
+@media (prefers-color-scheme: dark) {
+  :root {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+}
+```
+
+#### Putting it all together
+
+Combining all of the previous steps, we have a working userstyle!
```

Comment:
```suggestion
Combining all the previous steps, we have a working userstyle!
```

Reply:
I think this is just a preference with no correct answer. I'm more of a fan of the more explicit version with "of" as you can see but...